languages/js/src/helpers.ts

```diff
@@ -306,11 +306,10 @@ let RESET = '';
 let FG_BLUE = '';
 let FG_RED = '';
 if (
-  typeof window !== 'object' &&
-  typeof process.stdout.getColorDepth === 'function' &&
-  process.stdout.getColorDepth() >= 4 &&
-  typeof process.stderr.getColorDepth === 'function' &&
-  process.stderr.getColorDepth() >= 4
+  typeof process?.stdout?.getColorDepth === 'function' &&
+  process?.stdout?.getColorDepth() >= 4 &&
+  typeof process?.stderr?.getColorDepth === 'function' &&
+  process?.stderr?.getColorDepth() >= 4
```

Comment:
Once we know `process.stdxxx.getColorDepth` is a function, we don't need the optional chains 🙂 
```suggestion
  process.stdout.getColorDepth() >= 4 &&
  typeof process?.stderr?.getColorDepth === 'function' &&
  process.stderr.getColorDepth() >= 4
```